script/copy_deduplicate

```diff
@@ -135,6 +139,15 @@ parser.add_argument(
     default=0,
     help="Number of days preceding --date that should be used to filter out duplicates",
 )
+parser.add_argument(
+    "--billing_projects",
+    "--billing-projects",
+    "--billing_project",
+    "--billing-project",
+    nargs="+",
+    default=[None],
```

Comment:
Could we add a comment here indicating the significance of this default? It took me a while to realize the single None would be passed literally to `Client`. And probably also indicate in the help string what to expect if this is not set.